Plugins/ExaTrkX/include/Acts/Plugins/ExaTrkX/GraphStoreHook.hpp

```diff
@@ -9,24 +9,22 @@
 #pragma once
 
 #include "Acts/Plugins/ExaTrkX/ExaTrkXPipeline.hpp"
-#include "Acts/Plugins/ExaTrkX/detail/CantorEdge.hpp"
-#include "Acts/Utilities/Logger.hpp"
 
 namespace Acts {
 
-class TorchGraphStoreHook : public ExaTrkXHook {
+class GraphStoreHook : public ExaTrkXHook {
  public:
   using Graph = std::pair<std::vector<std::int64_t>, std::vector<float>>;
 
  private:
   std::unique_ptr<Graph> m_storedGraph;
 
  public:
-  TorchGraphStoreHook();
-  ~TorchGraphStoreHook() override {}
+  GraphStoreHook();
+  ~GraphStoreHook() override;
```

Comment:
is the destructor definition necessary or can it be implicitly defaulted?

Reply:
probably can be defaulted, but not removed..